dietpi/dietpi-software

```diff
@@ -14295,6 +14295,9 @@ _EOF_
 		# DietPi-Automation
 		if (( $G_DIETPI_INSTALL_STAGE == 0 )); then
 
+			# - Remove fake-hwclock, if real hwclock is available
+			hwclock && G_AGP fake-hwclock
```

Comment:
Minor, dev null this to prevent error print?
```
root@DietPi:~# hwclock && G_AGP fake-hwclock
hwclock: Cannot access the Hardware Clock via any known method.
hwclock: Use the --debug option to see the details of our search for an access method.
root@DietPi:~# hwclock &> /dev/null && G_AGP fake-hwclock
root@DietPi:~#
```

Reply:
@Fourdee 
Jep thought about this. Otherwise the result can be an explanation for end user why `fake-hwclock` is purged. Would be nice to use G_RUN_CMD here with manual, but I don't like the whiptail error menu always popping up.

Maybe we should add a G_RUN_CMD mode that just shows [FAILED] but does not show any further error message (github issue template) nor whiptail menu, thus we can use this to check commands error code, where we expect it may fail, e.g. also for NTP mirror check and such.
Okay it's also possible to &> /dev/null the output and use G_DIETPI-NOTIFY 2 to inform user about the check and the possible result (purging or leaving fake-hwclock in this case).

I will add comment with this for now.